ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -76,13 +76,21 @@ port_profiles:
     port_channel:
       description: < port_channel_description >
       mode: < "active" | "passive" | "on" >
-      # Allocates an automatic short_esi to all ports using this profile
-      # Please see the notes under "EVPN A/A ESI dual-attached endpoint scenario" before setting short_esi: auto.
-      short_esi: auto
+      # Please see the notes under "EVPN A/A ESI dual- and single-attached endpoint scenarios" before setting short_esi: auto
+      # Also note that defining short_esi under ethernet_segment overrides short_esi defined at the port_channel level
+      short_esi: < 0000:0000:0000 | auto >
```

Comment:
just some interrogation would it be better to use `xxxx:xxxx:xxxx` instead of `0000:0000:0000` to make it clear about the format? in `eos_cli_config_gen` the help states: `identifier: < EVPN Ethernet Segment Identifier (Type 1 format) >` but not sure it would be rendering nicely here